src/ui/server/routes/insurance/business/index.test.js

```diff
@@ -0,0 +1,20 @@
+import { get, post } from '../../../test-mocks/mock-router';
+import { ROUTES } from '../../../constants';
+
+import { get as getCompanyDetails, postCompaniesHouseSearch } from '../../../controllers/insurance/business/company-details';
+
+describe('routes/insurance/your-business', () => {
+  beforeEach(() => {
+    require('.'); // eslint-disable-line global-require
```

Comment:
Why do we have inline eslint rule please?

Reply:
removed